src/component/mxgraph/extension/MxScaleFactorCanvas.ts

```diff
@@ -111,4 +111,10 @@ export class MxCanvasUtil {
     const insetH = (shape.h - iconSize.height) / 2;
     c.translate(shape.x + insetW, shape.y + insetH);
   }
+
+  public static translateIconToShapeBottomCenter(canvas: mxgraph.mxXmlCanvas2D, shape: ShapeConfiguration, iconSize: Size): void {
+    const insetW = (shape.w - iconSize.width) / 2;
+    const insetH = shape.h - iconSize.height - 7;
```

Comment:
:question: where does the `-7` comes from? Is this something you found empirically when testing with various BPMN diagrams?

Reply:
Yes, I just test with different values.

Reply:
The spec states
> The Sub-Process marker MUST be a small square with a plus sign (+) inside. The square MUST be positioned at the bottom center of the shape.
A lot of BPMN vendor don't stick it to the bottom, so this seems valid

Reply:
@aibcmars the other flow using this marker will  be the call activity and we aim to reuse the current marker implementation